### PR TITLE
Dashboard

### PR DESCRIPTION
**Pull request by rrabinovitch**
TLDR: This PR covers the general set up of the Dashboard view without any styling and _starts_ to address subsequent user stories that the project board says we're supposed to work on together

Details:
* Spec directory organization:
     * moved login spec to its own file so that dashboard show spec just tests contents of that view
     * created umbrella feature tests directory to house subcategories of feature specs
* Add `Discover Movies` button, `friends` section, and `viewing-parties` section to dashboard - issue #18 
* `Discover Movies` button directs user to a discover page (associated with `DiscoverController#index` action) - issue #17 
* Expectations outlined for upcoming issues and user stories related to `friends` and `viewing-parties` sections, but view contents and related routes not yet complete

### CODE REVIEW

File: spec/features/welcome/login_spec.rb [hunk 1]
```diff
@@ -1,11 +1,10 @@
 RSpec.describe 'user can log in' do
   describe 'using google_oauth2', type: :feature do
-
-    it 'user authenticates via Google, and is redirected to their dashboard where they see Welcome, user_email' do
+    it "user authenticates via Google, and is redirected to their dashboard where they see 'Welcome, <user_email>'" do
```
Comment (gabichuelas) [inline spec/features/welcome/login_spec.rb:3]: @rrabinovitch good call on moving the log-in spec to it's own test file in the welcome folder. Makes more sense for sure!
